frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/stateUpdate/vmSettings/vm-settings-tab-state-update.ts

```diff
@@ -35,10 +35,6 @@ export const selectUserTemplateOnLoadedUpdater = (options: UpdateOptions) => {
   }
 
   const userTemplateName = iGetCommonData(state, id, VMWizardProps.userTemplateName);
-  if (!userTemplateName) {
```

Comment:
wrong return removed ^^ this return is quite useful.

Reply:
:man_facepalming:  :-)